Update to color properties in Forced Colors Mode

alisonmaher opened the pull request:
[css-color-adjust-1] This change makes the updates per resolutions taken in https://github.com/w3c/csswg-drafts/issues/11857.

More specifically, it moves the color properties that forced colors applies to within a note, mentions that this is the list at the time of writing and may not be exhaustive as other color properties are added over time, and notes that forced colors mode applies to any shorthand properties, as well.

It updates normative text referencing color properties to note that forced colors applies to the `<color>` components of all properties.

This change also updates `column-rule-color` to `rule-color` to ensure that full support exits for the new color properties added in the Gaps specification.

It adds `accent-color` to the list of color properties (it is mentioned lower down in the algorithm (along with `scrollbar-color`) but it wasn't mentioned in the color properties list).

I also updated some minor inconsistencies in using "computed" vs "computes" within the forced color mode rules.

Lastly, I updated the order of the color properties to be in alphabetical order.

The updates can be found hosted at https://alisonmaher.github.io/alisonmaher/css-color-adjust-1/Overview.html#forced-colors-properties.

svgeesus reviewed (approved):
LGTM

tabatkins commented:
Sorry, r-. Nothing wrong with the edits on their face, I just disagree with one of the resolutions this is implementing, and wrote my objection in #11857.

tabatkins reviewed (changes requested):
r- for now, until we resolve the issue

alisonmaher commented:
> Sorry, r-. Nothing wrong with the edits on their face, I just disagree with one of the resolutions this is implementing, and wrote my objection in #11857.

@tabatkins 
Updated the text to more clearly state that this applies to `<color>` components of properties, instead of "color properties".

I also noticed a couple of other minor things when making the update that are now fixed in this PR:
1. We used a mix of "computes" and "computed" in one of the lists in this section - I've updated all remaining instances to "computes"
2. `accent-color` was listed as computing to `auto` but wasn't included in the list of properties that are now in the note, so I added it there now, as well
